### ISSUE TITLE
Wrong URL using Windows and WebdriverIO v5

### ISSUE THREAD

**Issue by alippai** [webdriverio/webdriverio#2980]
## The problem

Running WebdriverIO v5 beta without wdio runner, if port is specified for localhost on windows I get errors:

```
2018-10-01T22:54:30.031Z INFO webdriver: [POST] http://0.0.0.0/:9515C/session
2018-10-01T22:54:30.031Z INFO webdriver: DATA { capabilities: { alwaysMatch: { acceptInsecureCerts: true } },
  desiredCapabilities: { alwaysMatch: { acceptInsecureCerts: true } } }
2018-10-01T22:54:31.067Z WARN webdriver: Request failed due to Error: connect ECONNREFUSED 0.0.0.0:80
2018-10-01T22:54:31.067Z INFO webdriver: Retrying 1/2
2018-10-01T22:54:31.068Z INFO webdriver: [POST] http://0.0.0.0/:9515C/session
2018-10-01T22:54:31.068Z INFO webdriver: DATA { capabilities: { alwaysMatch: { acceptInsecureCerts: true } },
  desiredCapabilities: { alwaysMatch: { acceptInsecureCerts: true } } }
2018-10-01T22:54:32.074Z WARN webdriver: Request failed due to Error: connect ECONNREFUSED 0.0.0.0:80
2018-10-01T22:54:32.075Z INFO webdriver: Retrying 2/2
2018-10-01T22:54:32.075Z INFO webdriver: [POST] http://0.0.0.0/:9515C/session
2018-10-01T22:54:32.075Z INFO webdriver: DATA { capabilities: { alwaysMatch: { acceptInsecureCerts: true } },
  desiredCapabilities: { alwaysMatch: { acceptInsecureCerts: true } } }
2018-10-01T22:54:33.080Z ERROR webdriver: Request failed after retry due to Error: Error: connect ECONNREFUSED 0.0.0.0:80
    at Request._callback (node_modules\webdriver\src\request.js:85:27)
    at self.callback (node_modules\request\request.js:185:22)
    at Request.emit (events.js:182:13)
    at Request.EventEmitter.emit (domain.js:442:20)
    at Request.onRequestError (node_modules\request\request.js:881:8)
    at ClientRequest.emit (events.js:182:13)
    at ClientRequest.EventEmitter.emit (domain.js:442:20)
    at Socket.socketErrorListener (_http_client.js:391:9)
    at Socket.emit (events.js:182:13)
    at Socket.EventEmitter.emit (domain.js:442:20)
    at emitErrorNT (internal/streams/destroy.js:82:8)
    at emitErrorAndCloseNT (internal/streams/destroy.js:50:3)
    at process._tickCallback (internal/process/next_tick.js:63:19)
```

If I start the very same command/test/script from WSL (Ubuntu linux), it works, finds the correct address.

## Environment

* WebdriverIO version: 5.0-beta.3
* Node.js version: 10.9.0
* Standalone mode or WDIO Testrunner: standalone

**Comment by StephenABoyd** [comment]
The issue is with the request.js file in the 'webdriver' package on the _createOptions method. It's using path.resolve() to build out the path for the url. This is causing it to add 'C:/' to the url.

**Comment by erwinheitzman** [comment]
Isn't this issue resolved by the merge of #2996?

**Comment by alippai** [comment]
I think it's resolved
